boot/boot.js

```diff
@@ -510,26 +536,47 @@ $tw.utils.evalGlobal = function(code,context,filename) {
 		contextValues.push(value);
 	});
 	// Add the code prologue and epilogue
-	code = "(function(" + contextNames.join(",") + ") {(function(){\n" + code + "\n;})();\nreturn exports;\n})\n";
+	code = `
```

Comment:
I think this type of templating isn't allowed in TW. TW is ES5 only atm.

Reply:
Oops, thanks. Fixed.